test_env.py

```diff
@@ -28,10 +28,15 @@
 sys.path.insert(0, os.getcwd()+"/..")
 
 __import__('sofagym')
-name = ['multigaitrobot-v0',
-        'gripper-v0', 'trunk-v0', 'trunkcup-v0', 'diamondrobot-v0',
-        'maze-v0', 'simple_maze-v0', 'concentrictuberobot-v0']
-num = 2
+name = ['multigaitrobot-v0',#[ERROR]   [RequiredPlugin(ModelOrderReduction)] Plugin not found: "ModelOrderReduction"
```

Comment:
I guess the comments are the errors you get. But they must not be in the code. I suggest to remove the comments